src/components/TodoApp.vue

```diff
@@ -165,6 +134,70 @@ function searchToDos(item: string) {
   searchQuery.value = item
 }
 
+function disactivateSort(isActive: boolean) {
+  isSortActive.value = isActive
+}
+
+function sortTodos(property: string, order: string, isActive: boolean) {
+  activeOrder.value = order
+  activeProperty.value = property
+  isSortActive.value = isActive
+  todos.value.sort((a: Todo, b: Todo) => {
+    switch (property) {
+      case 'Title':
+        return sortByTitle(a, b)
+      case 'Description':
+        return sortByDescription(a, b)
+      case 'Priority':
+        return sortByPriority(a, b)
+      case 'Date':
+        return sortByDate(a, b)
+      default:
+        return 0
+    }
+  })
+}
+
+function sortByTitle(a: Todo, b: Todo) {
+  let fa = a.title.trim().toLowerCase(),
```

Comment:
Add more descriptive variable name, also dont forget what we talked about declaring multiple variables